docs/proc/code_contribution_guide.md

```diff
@@ -0,0 +1,31 @@
+# Bref
+As team we should follow rules to have understandable processes and code. This document covers how we work with versions, codes and its styles. If repository does not contain own rules, then this rules SHOULD be used.
+
+Also please pay attention that some repositories has more complicated rules, these rules have own sections inside this document.
+
+# Code style guide
+
+- Rust: we respect (rust code style guide)[https://doc.rust-lang.org/nightly/style-guide/] and we should follow it. Exception: if repository has own code style guide (ex: (polkadot-sdk)[https://github.com/paritytech/substrate/blob/master/docs/STYLE_GUIDE.md]) we should use it.
+- JS: we respect (Google JS Style guide)[https://google.github.io/styleguide/jsguide.html]
+- Bash/Python/Other languages: please remember about basic styles and try to do not reach 80 columns limit.
+
+# Develop and version guide
+
+## General develop and version guide
+- Each new feature or a fix SHOULD have a branch name with ticket number. Format: issue/NUMBER. If you have a huge ticket and dont have issue, please create a ticket.
```

Comment:
> Format: issue/NUMBER

So we can have collisions when 2 and more devs working on same issue.
Maybe `$username/issue-$number` ?

Reply:
So branch management is easier, you can understand who is branch owner by its name.

Reply:
Make sense, let me update it.

Reply:
Updated.